ruby_connectors/lib/ruby_connectors/temporal/workflows/fetch_schema_workflow.rb

```
@@ -0,0 +1,27 @@
+            backoff: 2,
+            max_attempts: 3
+          }
+
+          activity do |opts|
+            opts.retry_policy = retry_policy
+            RubyConnectors::Temporal::Activities::FetchSchemaActivity.execute!(connector_id)
+          end
+        rescue StandardError => e
+          logger.error("Failed to fetch schema for connector #{connector_id}: #{e.message}")
+          raise
+        end
```

rubocop

[convention] Method has too many lines. [13/10]

(Metrics/MethodLength)